### DISCUSSION TITLE
我在两地都搭建了ppgw，用zerotier回家，一个能返回fakeip，另一个返回的是正常IP，可能是什么原因

### DISCUSSION THREAD

**Discussion by jq88kazze** [kkkgo/PaoPaoGateWay#85] [问和答]
![IMG_0339](https://github.com/kkkgo/PaoPaoGateWay/assets/24644040/004ef134-975d-4bb0-95c1-d83473d04efe)

**Comment by kkkgo** [comment]
跟PaoPaoDNS和PaoPaoGateway无关，建议检查IP的上游DNS。

**Reply by kkkgo** [reply]
zerotier的话，可以看看是哪个进程在监听53端口提供的结果。
